ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -1132,6 +1158,20 @@ gst_tensor_src_tizensensor_fill (GstBaseSrc * src, guint64 offset,
   GstMemory *mem;
   GstMapInfo map;
 
+  if (self->freq_n > 0) {
+    gulong usec;
+    struct timespec t;
+
+    clock_gettime (CLOCK_MONOTONIC, &t);
+    g_assert (self->freq_d > 0);
+
+    usec = ((gulong) self->freq_d * 1000000LU) / self->freq_n;
```

Comment:
How about using `G_USEC_PER_SEC` for code readability?

Reply:
Applied!